x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -222,8 +221,8 @@ public final void test() throws Throwable {
              * The csv tests support all but a few features. The unsupported features
              * are tested in integration tests.
              */
-            assumeFalse("metadata fields aren't supported", testCase.requiredFeatures.contains(EsqlFeatures.METADATA_FIELDS.id()));
-            assumeFalse("enrich can't load fields in csv tests", testCase.requiredFeatures.contains(EsqlFeatures.ENRICH_LOAD.id()));
+            assumeFalse("metadata fields aren't supported", testCase.requiredCapabilities.contains("metadata_fields"));
```

Comment:
Nit: using constants could be more robust to future changes. We could make `EsqlCapabilities.cap` public, then this could just be
```suggestion
            assumeFalse("metadata fields aren't supported", testCase.requiredCapabilities.contains(cap(EsqlFeatures.METADATA_FIELDS)));
```

Reply:
Yeah, I probably should do that.